### DISCUSSION TITLE
Does the Deepgrams's Speech-to-Text uses Transfer Learning?

### DISCUSSION THREAD

**Discussion by umarmirza0341** [deepgram/community#888] [General help] [answered]
We are currently working on a proof of concept and have selected the Deepgram Nova2 model for transcribing our MP3 audio files. We are testing the model's accuracy and have found that the files that are transcribed give better accuracy as we move from start to end. Does anyone know that Deepgram uses the transfer Learning in its models?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@umarmirza0341  Let me ask a few people and see if i can get you answer.

**Comment by jpvajda** [comment, marked as answer]
@umarmirza0341 After chatting with a few Engineers:

We do use transfer learning in our models, but we don't believe this is relevant to what you are seeing.

 For a long audio file we don't know why later points in the audio would be overall better from your perspective, but our guess would be it is a function of the audio (_as an example: a nervous person talking gets more confident/clearer as they go along_).  But transfer learning doesn't have anything to do with what you are seeing in this case.

Also when you say "accuracy" what specifically are you referring to? How are you measuring accuracy in this case.
